src/nvim/globals.h

```diff
@@ -724,7 +724,9 @@ EXTERN bool need_highlight_changed INIT(= true);
 
 EXTERN FILE *scriptout INIT(= NULL);  ///< Stream to write script to.
 
-// volatile because it is used in a signal handler.
+// no need volatile because neovim does not handle SIGINT.
+// Note that even when handling SIGINT, volatile is not necessary because the
+// callback is not called directly from the signal handlers.
 EXTERN bool got_int INIT(= false);          // set to true when interrupt signal occurred
```

Comment:
```suggestion
// Note that even when handling SIGINT, volatile is not necessary because the
// callback is not called directly from the signal handlers.
EXTERN bool got_int INIT(= false);          // set to true when interrupt signal occurred
```